net/bandwhich/Makefile

```diff
@@ -0,0 +1,91 @@
+# This is a Framework Template for a Rust-Lang enabled OpenWrt Package.
+#
+# Written: Donald Hoskins <grommish@gmail.com>, 2022
+#
+# Instructions:
+#
+# This assumes a Github-based Repository
+#
+# 1) Fill in The Blanks <>
+# 2) Build/Compile is used to call cargo and/or rustc - See the below
+#    example on how to use the variables and format
+# 3) Save the Makefile, and run the following to update the feeds index:
+#    ./scripts/feeds update -i
+#    ./scripts/feeds install <pkg_name>
+# 4) Call 'make package/feeds/packages/to/app/{download,check} FIXUP=1'
+# 5) cargo packages typically do not have a ./configure step, so you
+#    can leave that entire section blank.
+# 6) Build/Compile is used to call cargo and/or rustc - See the below
+#    example on how to use the variables and format
+# 7) Be sure to run `make menuconfig` PRIOR to building, so that it updates the
+#    dependencies.
+#
+#
+# Notable Variables Used:
+# CARGO_HOME - Sysroot where cargo/rustc is installed
+# RUSTC_HOST_ARCH - The GNU/LLVM Tuple of the Build Host
+# RUSTC_TARGET_ARCH/REAL_GNU_TARGET_NAME - The GNU/LLVM Tuple of the Target Device
+# RUSTFLAGS - Additional Build Flags used by the Rust compiler
+# TARGET_CONFIGURE_OPTS - Used to add CARGO_HOME Environmental
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := bandwhich
+PKG_VERSION := 0.20.0
+PKG_RELEASE := 1
+
+PKG_SOURCE_PROTO := git
+PKG_SOURCE_DATE := 2020-10-15
+PKG_SOURCE_VERSION := a703513412d4da71085e55fd9d62a080527b2800
+PKG_SOURCE_URL := https://github.com/imsnif/bandwhich.git
+PKG_MIRROR_HASH := 647e1412a84b224a2490c316fc536ee838dab1957d4c5bfb1dbdfe49b870d8d9
+
+PKG_BUILD_DEPENDS := rust/host
```

Comment:
You can remove the whitespace between ``:=``

Reply:
Is there a reason for this other than looks?  `nano` syntax highlights for GNU Makefile, but only when it's properly spaced.

Reply:
Well, you are doing something here special. You might want to take a look at it. ;-) All the Makefiles here does not have it.

Reply:
I've made the changes, but I did it that way because of the Reference https://www.gnu.org/software/make/manual/make.html#Reference that outlines the Simple Expanded Variables.  I'll keep it in mind for the future though.  I never bothered to check the spacing in previous PR 👍 

This isn't going anywhere until I can get traction on testing rust across the arches, but I'm hoping to generate at least some interest if it's more than a "hypothetical"

Reply:
Removed extraneous whitespace